tests/plugins/tool/black_tool_plugin/valid_package/format_errors.py

```diff
@@ -1 +1 @@
-x =  "some string",
+x = ("some string",)
```

Comment:
I think this change needs to be reverted. We don't want to run `black` on this file as we have intentional errors to trigger issues for unit tests. I expect a unit test will fail due to this change.